File: php/lecture/classStruct/Titik.php
<?php
class Titik{
    private $x;
    private $y;

    public function __construct(){
        $this->x = 0;
        $this->y = 0;
    }

    public function setX($xp){
        $this->x = $xp;
    }

    public function getX(){
        return $this->x;
    }

    public function setY($yp){
        $this->y = $yp;
    }

    public function getY(){
        return $this->y;
    }

    function __destruct(){
    }
}
?>

File: Cpp/classStruc/Titik.cpp
class Titik{
    private:
        int x;
        int y;
    
    public:
        Titik(){
            x = 0;
            y = 0;
        }

        Titik(int xp, int yp){
            x = xp;
            y = yp;
        }

        void setX(int xp){
            x = xp;
        }

        int getX(){
            return x;
        }

        void setY(int yp){
            y = yp;
        }

        int getY(){
            return y;
        }

        ~Titik(){}
};